fischkapp/src/components/styles/Header.styled.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from "styled-components";
+
+export const StyledHeader = styled.div`
+  width: 100%;
+  height: 9vh;
+  position: fixed;
+  padding: 2.5vh;
+  box-shadow: 0px -16px 52px -10px rgba(152, 163, 207, 1);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .header__logo {
```

Comment:
**nitpick**: You should avoid such deep nesting of styles. Let's split it into smaller styled-components